youtube_dl/extractor/funimation.py

```diff
@@ -215,9 +245,12 @@ def _real_extract(self, url):
         self._sort_formats(formats)
 
         title = item['title']
-        artist = item.get('artist')
-        if artist:
-            title = '%s - %s' % (artist, title)
```

Comment:
>Also fixed the returned video information to use the 'series' key rather than the 'artist' key, the latter of which is meant for albums.

`artist` was not used in returned info dict. Local naming schema does not matter.